src/features/ui/Wrapper/Wrapper.module.scss

```diff
@@ -0,0 +1,10 @@
+.Container {
+	.LabelContainer {
+		display: flex;
+		justify-content: space-between;
+
+		.Label {
+			margin-left: 1.375rem;
```

Comment:
Could use gap over margins here I think.

Reply:
Doesn't seem to apply the left spacing if I use `gap` on `LabelContainer` here. Any suggestions?